src/Template/rector.php

```diff
@@ -36,7 +36,7 @@
 use Rector\Set\ValueObject\SetList;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->sets([SetList::DEAD_CODE, LevelSetList::UP_TO_PHP_74, PHPUnitSetList::PHPUNIT_SPECIFIC_METHOD, PHPUnitSetList::PHPUNIT_80]);
+    $rectorConfig->sets([SetList::DEAD_CODE, LevelSetList::UP_TO_PHP_74, PHPUnitSetList::PHPUNIT_SPECIFIC_METHOD, PHPUnitSetList::PHPUNIT_91]);
```

Comment:
The PHPUnitLevelSetList::UP_TO_PHPUNIT_100 should can be used so cover from all previous version 

https://github.com/rectorphp/rector-phpunit/blob/26e1dcbc5fa3d214af34f627d0d7013b8eba8eb5/src/Set/PHPUnitLevelSetList.php#L39

Reply:
Oo nice! I will update.

Reply:
Done!